websites/C/Cuevana 3/presence.ts

```diff
@@ -30,40 +30,103 @@ presence.on(
 );
 
 presence.on("UpdateData", async () => {
+	if (document.location.hostname === "ww1.cuevana3.me") {
+		const presenceData: PresenceData = {
+			largeImageKey: "logo",
+		};
+
+		if (
+			document.location.pathname.match("^/[0-9]") ||
+			document.location.pathname.includes("/episodio")
+		) {
+			const titulo = document.querySelector("h1.Title").textContent,
+				subtitulo = document.querySelector("h2.SubTitle").textContent,
+				cover = document
+					.querySelector("div.backdrop > article > div.Image > figure > img")
+					.getAttribute("data-src");
+
+			if (!document.location.pathname.includes("/episodio")) {
+				presenceData.details = titulo;
+				if (subtitulo === titulo) delete presenceData.state;
+				else presenceData.state = subtitulo;
+				presenceData.buttons = [
+					{ label: "Ver Película", url: window.location.href },
+				];
+				presenceData.largeImageKey = cover;
+			} else {
+				presenceData.details = titulo
+					.replace(document.querySelector("h1.Title > span").textContent, "")
+					.replace(/\s+/g, " ")
+					.trim();
+				presenceData.state = `${
+					document.querySelector("h1.Title > span").textContent
+				} ${subtitulo}`;
+				presenceData.buttons = [
+					{ label: "Ver Episodio", url: window.location.href },
+				];
+				presenceData.largeImageKey = cover;
+			}
+
+			if (iFrameVideo) {
+				(presenceData.smallImageKey = videoPaused ? "pause" : "play"),
+					(presenceData.smallImageText = videoPaused
+						? (await strings).pause
```

Comment:
Use the Awaited type for strings and remove the await here as shown below
https://github.com/PreMiD/Presences/blob/5e5f99668ba4800524e66c2a3fc9fd38755025c6/websites/G/Giggl/presence.ts#L19